packages/ozone/src/config/env.ts

```diff
@@ -13,6 +13,9 @@ export const readEnv = (): OzoneEnvironment => {
     pdsDid: envStr('OZONE_PDS_DID'),
     dbPostgresUrl: envStr('OZONE_DB_POSTGRES_URL'),
     dbPostgresSchema: envStr('OZONE_DB_POSTGRES_SCHEMA'),
+    dbPoolSize: envInt('OZONE_DB_POOL_SIZE'),
```

Comment:
these crept in cause I thought i was running into some connection pool issues in the test, but it turns out that wasn't the case

kept them around since it's just nice to have 👍